Week7/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/model/response/RemoteMovieResponse.kt

```diff
@@ -0,0 +1,37 @@
+package com.raywenderlich.myfavoritemovies.model.response
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class MovieTopLevelResponse(val data: MoviesResultResponse, val about: AboutResultResponse)
+
+@Serializable
+data class AboutResultResponse(val version: String, val serverTime: String)
+
+@Serializable
+data class MoviesResultResponse(val movies: Array<MovieResponse>) {
```

Comment:
Not sure why you used equals if you have `data` modifier for the classes, as equals is automatically generated.